Corona-Warn-App/src/main/res/layout/contact_diary_edit_locations_fragment.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="de.rki.coronawarnapp.contactdiary.ui.edit.ContactDiaryEditLocationsViewModel" />
+
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/content_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.appcompat.widget.Toolbar
+            android:id="@+id/toolbar"
+            android:layout_width="0dp"
```

Comment:
please check any layout_width="0dp" to use @dimen/match_constraint which is 0dp

Reply:
There is no point in using a dimen for this since 0dp will never change.

Reply:
i like it, i'd like to maybe refactor all the toolbars into a common one on activity level later when there is time 👍

Reply:
`@dimen/match_constraint` is not supposed to change, its supposed to make it more obvious for people reading the code that the width will match the constraint layout parameters

Reply:
You are right but it is just for semantiv reason. And as Koyla already pointed out it helps others to better unterstand what you're trying to achieve. Ofc no deal breaker

Reply:
I think @d4rken asked not to use that anymore, or not?